dynacred/src/Data.ts

```diff
@@ -222,7 +225,7 @@ export class Data {
         return DarcInstance.fromByzcoin(bc, signer[0]);
     }
 
-    static async attachDevice(url: string): Promise<Data> {
+    static async attachDevice(config: Config, url: string): Promise<Data> {
```

Comment:
same thing as in `connectByzcoin` - I prefer using the interpretet config here as `bc: ByzCoinRPC`.